src/main/kotlin/calculator/domain/AddCalculator.kt

```diff
@@ -0,0 +1,14 @@
+package calculator.domain
+
+class AddCalculator {
+    private var _sumResult: Int = 0
+    val sumResult: Int
+        get() = _sumResult
```

Comment:
```suggestion
    var sumResult: Int = 0
    private set
```

backingProperty 사용도 좋지만, 이 경우에는 private set으로 수정에 대한 부분을 외부에서 불가능하도록 만드는 게 좋지 않을까요?

Reply:
같은 효과를 복잡하게 생각했네요 :)
private set 이 단순하고 좋은것 같습니다.